app/javascript/styles/components.scss

```diff
@@ -3061,6 +3061,33 @@ button.icon-button.active i.fa-retweet {
   background: $base-overlay-background;
 }
 
+.media-modal__pagination {
+  width: 100%;
+  text-align: center;
+  position: absolute;
+  left: 0;
+  bottom: -40px;
+
+  li {
+    display: inline-block;
+  }
+
+  button {
```

Comment:
you should generally avoid nested selectors in your css. 
instead, you should use an element class, like `.media-model__pagination_button` (or something less wordy :D)